variables.json

```diff
@@ -117,12 +117,49 @@
     "schematron_filter": "validations/qdcingest_reqd_fields.sch",
     "schematron_report": "validations/padigital_missing_thumbnailURL.sch"
   },
-  "PENNSTATE_TARGET_ALIAS_ENV": "dev",
-  "PENNSTATE_XSL_CONFIG": {
+  "USCRANTON_TARGET_ALIAS_ENV": "dev",
```

Comment:
this seems to be writing over pennstate xsl config? I'm confused. Also, if Scranton is csv, there will be no OAI config. The variables would look more like what we have for Free Library.

Reply:
Only the Internet Archive collection is CSV; the rest are from cdm

Reply:
They only have one CSV collection. Let me fix this.